test/invalid.js

```diff
@@ -129,6 +129,14 @@ describe("Error object structure", () => {
     const { level } = validation[0];
     expect(level).toBe("warning");
   });
+  it("allows `instanceof WebIDLParseError`", () => {
+    try {
+      parse("throwerror");
+      throw new Error("Shouldn't reach here");
+    } catch ({err}) {
```

Comment:
`{err}` accesses `err` field of the error object, I think we just want `err` to get the error object itself.

Reply:
Okay I have made the changes and they have passed. Thanks for your help :smile: